src/task_manager/dive_state.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import roslib
+
+import rospy
+import smach
+import smach_ros
+
+from robosub.DiveAction.msg import DiveAction
+from robosub.DiveGoal.msg import DiveGoal
+
+from actionlib import *
+from actionlib_msgs.msg import *
+
+
+class Dive_State:
```

Comment:
This is failing pylint because it requires that you specify a parent class for all classes. You can write `class Dive_State(object)` instead